MUXSDKStats/MUXSDKStats/MUXSDKStats.m

```diff
@@ -231,6 +231,7 @@ + (void)programChangeForPlayer:(nonnull NSString *)name withVideoData:(nullable
     [MUXSDKStats videoChangeForPlayer: name withVideoData:videoData];
     MUXSDKPlayerBinding *player = [_viewControllers valueForKey:name];
     if (player) {
+        [player monitorAVPlayerItem];
```

Comment:
This works and my QA is showing the program change correctly as 2 different views. 
1. `MUXSDKStats:programChangeForPlayer` is called
1. `MUXSDKStats:videoChangeForPlayer` updates the cached video metadata
1. `MUXSDKPlayerBinding:monitorAVPlayerItem` results in a call to `MUXSDKPlayerBindingManager:videoChangedForPlayer` which dispatches `viewinit` and `Data Event` with updated metadata
1. We manually dispatch `play` and `playing` since in this case KVO doesn't fire

My only comment is that perhaps we should add a method `programChangedForPlayer` on `MUXSDKPlayerBinding` and have it call `monitorAVPlayerItem` `dispatchPlay` and `dispatchPlaying` itself - so we don't need to expose `monitorAVPlayerItem` in the public header.

Reply:
I like that suggestion. I'm going to try to add a test for this too so I'll do that